tests/browserstack_automation/page_objects/ready.page.js

```diff
@@ -63,6 +65,44 @@ class ReadyPage extends Page {
     return $$('//*[contains(@id, "readyFinePrintStepText")]');
   }
 
+  get finePrintStepHeaderText1()
+  {
+    return $("(//div[@class='StepTitle-sc-lvvjo6-10 iKrjzD'][normalize-space()='You cannot cast your vote electronically'])[1]");
+  }
+
+  get finePrintStepHeaderText2()
+  {
+    return $('//div[text() = "WeVote does not represent a government entity"]');
+  }
+
+  get finePrintStepHeaderText3()
+  {
+    return $('//div[text() = "Please make sure you are registered to vote"]');
+  }
+
+  get finePrintStepHeaderText4()
+  {
+    return $('//div[text() = "How your data is used & protected"]');
+  }
+
+  async checkFinePrintHeaders()
+  {
+    const text1 = (await this.finePrintStepHeaderText1);
+    console.log("text1: " + text1);
+    const text2 = (await this.finePrintStepHeaderText2);
+    console.log("text2: " + text2);
+    const text3 = (await this.finePrintStepHeaderText3);
+    console.log("text3: " + text3);
+    const text4 = (await this.finePrintStepHeaderText4);
+    console.log("text4: " + text4);
+
+    await expect(text1).toHaveText('You cannot cast your vote electronically');
```

Comment:
Could you modify this test case to look if element is displayed instead of searching for secific text such as if id is present